python/servo/try_parser.py

```diff
@@ -192,6 +214,43 @@ def test_empty(self):
                               }
                               ]})
 
+    def test_job_merging(self):
+        self.assertDictEqual(json.loads(Config("wpt-2020 wpt-2013").to_json()),
+                             {'fail_fast': False,
+                              'matrix': [{
+                                  'name': 'Linux WPT',
+                                  'profile': 'release',
+                                  'unit_tests': False,
+                                  'workflow': 'linux',
+                                  'wpt_layout': 'all',
+                                  'wpt_tests_to_run': ''
+                              }]
+                              })
+
+        # These two jobs should not be merged to one that turns on unit tests.
+        a = JobConfig("Linux", Workflow.LINUX, unit_tests=True)
+        b = JobConfig("Linux", Workflow.LINUX, unit_tests=False)
+        self.assertTrue(a.merge(b))
+        self.assertEqual(a, JobConfig("Linux", Workflow.LINUX, unit_tests=True))
```

Comment:
I think the comment is wrong here as jobs are merged to one, fight?

Reply:
I think my comment was written in a pretty confusing way. I've removed all these comments and converted them to assert messages, which is more useful, I think.